application-numbered-content-headings/application-numbered-content-headings-toc/src/main/java/org/xwiki/contrib/numbered/content/toc/TocTreeBuilder.java

```diff
@@ -166,6 +166,9 @@ private Block generateTree(List<HeaderBlock> headings, int start, int depth, Str
             tocBlock = currentBlock.getRoot();
         }
 
+        /** Add CSS class to ease styling. */
+        tocBlock.setParameter("class", "wikitoc");
```

Comment:
This causes a null pointer exception when the toc is empty.

Reply:
I guess it should be moved to to if block just above, similarly to https://github.com/xwiki/xwiki-rendering/pull/232/files#diff-1d840a580cb113034422c099312b4b4145142302e341f2d4a78692f43aa8c3d8R151-R153

Reply:
Thank you both, I had missed that important change upstream, now fixed.